remove.php

```diff
@@ -1,3 +1,53 @@
 <?php
 //hidden from users
-//TODO: remove an item from comp, but only if the comp hasn't started yet.
\ No newline at end of file
+//TODO: remove an item from comp, but only if the comp hasn't started yet.
+
+
+$servername = "localhost";
+$username = "root";
+$password = "billybob";
+$dbname = "test";
+
+
+if (!isset($_POST['name']) || !isset($_POST['id'])) {
+    die("Missing 'name' or 'id' parameters.");
+}
+
+//Initialize name and id
+$name = $_POST['name'];
+
+
+
+
+
+try {
+    $pdo = new PDO("mysql:host=$servername;dbname=$dbname", $username, $password);
+    $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+    // Sanitize and validate the input
+   
+    $name = filter_var($_POST['name'], FILTER_SANITIZE_STRING);
+    $redirect = filter_var($_POST['redirect'], FILTER_SANITIZE_URL);
+    $postId = filter_var($_POST['id'], FILTER_SANITIZE_STRING);
```

Comment:
This is deprecated, we don't need to use it if we use PDO bindparam

Reply:
Roger

Reply:
Looks like I was wrong, but we should use filter_input instead because it can also replace the isset statements as well.